queries/menhir/highlights.scm

```diff
@@ -0,0 +1,29 @@
+["%parameter" "%token" "%type" "%start" "%attribute" "%left" "%right" "%nonassoc" "%public" "%inline" "%prec"] @keyword
+["%on_error_reduce"] @exception
+
+["let"] @keyword.function
```

Comment:
does let start a function or just a grammar rule?

Reply:
It's just a grammar rule.
There are no functions in menhir (well, you can have parameterised grammar rules which kinda look like functions but in the end it's a grammar rule).
Do you think `@keyword` would be more appropriate?

Reply:
it should be ok